packages/trpc/server/routers/viewer/availability.tsx

```diff
@@ -201,9 +201,16 @@ export const availabilityRouter = router({
         const schedule = await prisma.schedule.create({
           data,
         });
-        const hasDefaultScheduleId = await hasDefaultSchedule(user, prisma);
-        if (!hasDefaultScheduleId) {
-          await setupDefaultSchedule(user.id, schedule.id, prisma);
+
+        if (!user.defaultScheduleId) {
+          await prisma.user.update({
+            where: {
+              id: user.id,
+            },
+            data: {
+              defaultScheduleId: schedule.id,
+            },
+          });
```

Comment:
This function returns the first schedule as the default if the user does not have one set. If this is the case then we should set that schedule as the default

Reply:
Isn't this what setupDefaultSchedule did/does? 😕

Reply:
That's what I thought as well, but it wasn't triggering

Reply:
Dug into it some more. It's because the getter always an id, either the user default id or the first schedule in the query. That means the update never triggered.